### ISSUE TITLE
Library was not correctly initialized in Arctic

### ISSUE THREAD

**Issue by tomnewg** [man-group/arctic#965]
Hi guys,

I have been using the free mongodb version for over a year together with arctic and everything was working fine.
I recently upgraded my mongodb from a shared to a dedicated service.
From my personal machine and a server everything is running f.
I have a second server and I get the following error message
arctic.exceptions.LibraryNotFoundException: Library X was not correctly initialised in <Arctic at xxxxx...
.. the error message goes on talking about SSL: CERTIFICATE_VERIFY_FAILED

I have the server IP on the white list. The weird thing is the code is working from my laptop and the other server but not on the second server i.e. I can rule out something is wrong with the code or the databases in mondodb. 

Any idea what I could do? 

Many thanks

Tom

**Comment by tomnewg** [comment]
I don't think it is worth spending time on this issue. I moved the database back to a shared one

**Comment by dunckerr** [comment]
Hi Tom - can you give us a precise description of your environment, old and new please?  from the error message snippet  in your email I'd guess you have a credentials problem.  Probably worth checking you can login to the new server from wherever you're running  the python.
Cheers
Duncan

On Jul 4, 2022, at 4:37 PM, Tom Neugebauer ***@***.***> wrote:

﻿

Closed #965<https://github.com/man-group/arctic/issues/965> as completed.

**Comment by tomnewg** [comment]
Hi @dunckerr 

After downsizing the mongodb to the exact setting it was before I still get this error message.
I can run the script (I even created one just connecting and selecting a database) from anywhere else just not this server (it used to work up until 2 weeks ago).
The server IP is on the IP access list. I was told nothing has changed with this server.

What information about the environment do you need?

Cheers

Tom
